### PR TITLE
Here I have added some windows keyboard shortcuts

### PR DESCRIPTION
**Pull request by amanaggarwal06**
### Checklist:
- [x] Read our [contributing guidelines](../docs/CONTRIBUTING.md).
- [x] Search for duplicates.
- [x] Used an informative name for this pull request.

### Follow-up
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by amanaggarwal06** [comment]
Hey, I just created the pull request. 
Kindly check it out.

File: CheatSheets/windows-cheatsheet.md [hunk 1]
```diff
@@ -35,6 +35,24 @@
 | `Ctrl + O` | Open |
 | `Ctrl` + `+` | to increase the size |
 | `Ctrl` + `-` | to decrease the size |
+|` Ctrl + F4`| Close the active document |
+|` Ctrl + D`| Delete the selected item and move it to the Recycle Bin|
+|` Ctrl + E`| Open Search(in most apps)|
+|` Ctrl + R`| To refresh the active window|
+|` Ctrl + Y`| Redo an action|
```
Comment (crescentpartha) [inline CheatSheets/windows-cheatsheet.md:42]: This command already exists in the `General Shortcuts` section.

File: CheatSheets/windows-cheatsheet.md [hunk 2]
```diff
@@ -46,6 +64,15 @@
 | `Win + E` | Open File Explorer |
 | `Win + ,` | see your desktop until you hold keys |
 | `Win + v` | to see clipboard and emoji |
+| `Win + .`| Open Emojis section |
```
Comment (crescentpartha) [inline CheatSheets/windows-cheatsheet.md:67]: This command already exists in the `App Rearrangement Shortcuts (Metro)` section.

File: CheatSheets/windows-cheatsheet.md [hunk 3]
```diff
@@ -46,6 +64,15 @@
...
+|` Win + Upper Arrow key`| Maximize windows|
+|` Win + X`| Open the Quick Link menu|
+|` Win + L`| Lock your PC|
+|` Win + PrtScn`| For taking screenshot|
+|` Fn + Win + Spacebar`| For taking screenshots|
```
Comment (crescentpartha) [inline CheatSheets/windows-cheatsheet.md:72]:
```suggestion
| ` Win + PrtScn` <br /> ` Fn + Win + Spacebar` | For taking screenshots |
```
Please, `merge these commands` in one like that.

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Everything is Ok, What you have to do, is just delete the duplicate commands and rearrange all commands according to most required uses.

File: CheatSheets/windows-cheatsheet.md (hunk 1 above)
Comment (amanaggarwal06) [reply CheatSheets/windows-cheatsheet.md:42]: Okay, sorry for that.
Now I'll resolve this.

**Pull request closed without merging**